assignments/array-methods.js

```diff
@@ -78,8 +86,26 @@ console.log(ticketPriceTotal);
 // ==== Challenge 5: Be Creative ====
 // Now that you have used .forEach(), .map(), .filter(), and .reduce().  I want you to think of potential problems you could solve given the data set and the 5k fun run theme.  Try to create and then solve 3 unique problems using one or many of the array methods listed above.
 
-// Problem 1
+// Problem 1 Need a party invite list for participants who donated over 200
+let partyList = [];
+runners.map(function(info) {
+  if (info.donation > 200)
+  return partyList.push(`${info.first_name} ${info.last_name} should be added to the guest list`);
```

Comment:
Here it would be good practice to save `runners.map(...)` in a variable and reassign `partyList` like so: `partyList = guestListRunners`. Using `.push()` inside of a `.map()` isn't the best method 👍